package zamyslov.centreit.testtask.exception;

public class RecordNotFoundException extends Exception {
    public RecordNotFoundException(String msg) {
        super(msg);
    }
}
